src/esockd_acceptor.erl

```diff
@@ -171,48 +202,49 @@ handle_event(
 ) ->
     %% make it look like gen_tcp:accept
     inet_db:register_socket(Sock, SockMod),
-
-    %% Inc accepted stats.
-    esockd_server:inc_stats({Proto, ListenOn}, accepted, 1),
-
     case eval_tune_socket_fun(TuneFun, Sock) of
-        {ok, Sock} ->
-            case esockd_connection_sup:start_connection(ConnSup, Sock, UpgradeFuns) of
+        {ok, NewSock} ->
+            case start_connection(ConnSup, NewSock, UpgradeFuns) of
                 {ok, _Pid} ->
+                    %% Inc accepted stats.
+                    inc_stats(State, accepted),
```

Comment:
why move to here?  I think the old code looks fine. the socket is indeed accepted isn't it?

Reply:
there was previously no clear definition what `accepted` means.
and there was no counters for error cases.

now it means: accepted from the listener backlog, and successfully started an owner process.
and this counter is disjoint from error cases.